src/ducks/Chart/Signals/helpers.js

```diff
@@ -134,6 +137,17 @@ export function makeSignalDrawable (
 export const checkPriceMetric = metric => metric === Metric.price_usd
 
 export const AlertBuilder = {
-  price_usd: buildPriceSignal,
-  daily_active_addresses: buildDAASignal
+  price_usd: (slug, value, type) => {
+    return buildValueChangeSignal(slug, value, type, Metric.price_usd)
+  },
+  daily_active_addresses: buildDAASignal,
+  social_volume_total: (slug, value, type) => {
+    return buildValueChangeSignal(slug, value, type, Metric.social_volume_total)
+  },
+  exchange_balance: (slug, value, type) => {
+    return buildValueChangeSignal(slug, value, type, Metric.exchange_balance)
+  },
+  transaction_volume: (slug, value, type) => {
+    return buildValueChangeSignal(slug, value, type, Metric.transaction_volume)
+  }
```

Comment:
Metric could be passed as 4th argument to the mapped function here:
https://github.com/santiment/app/blob/e0ea4c6c38f85885f393e068534f908bb612a04c/src/ducks/Chart/Signals/index.js#L105
This will allow to assign `buildValueChangeSignal` as a reference and reduce repetitive code.